blazar/tests/plugins/oshosts/test_physical_host_plugin.py

```diff
@@ -184,7 +184,8 @@ def setUp(self):
     def reservation_allocation_dict(self, r_id, l_id, p_id, h_ids):
         return {
             'id': r_id, 'status': 'active', 'lease_id': l_id,
-            'start_date': '2015-01-01', 'end_date': '2015-01-02',
+            'start_date': datetime.datetime(2015, 1, 1, 0, 0),
+            'end_date': datetime.datetime(2015, 1, 2, 0, 0),
```

Comment:
does this patch change the allocation start-date and end-date from a string to a datetime object?

Reply:
No it's always been a datetime, but since everything was mocked, it never actually mattered for the tests. Since we are now adding `timedelta`, it does matter.